recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -59,5 +59,17 @@ do_install () {
 }
 
 pkg_postinst_ontarget_${PN} () {
+	# Add the memory reservation for Base. We assert that if the section exists,
+	# it's already been added.
+	if ! grep -q 'RtLinuxMemReserve' /etc/natinst/share/ni-rt.ini; then
+		cat >> /etc/natinst/share/ni-rt.ini <<EOF
+[RtLinuxMemReserve]
+Base=24
+EOF
```

Comment:
What workflow is this setting actually enabling? Is there a more specific package that it can go into?

The comment seems like a bad reason to not use `nirtcfg` to set this setting. Unless the original RB makes some compelling argument why we expect this value to exist prior to base image install, I think we can change this to `nirtcfg --set`.